### DISCUSSION TITLE
Blazor Server - How should I track dependencies (like EFCore)?

### DISCUSSION THREAD

**Discussion by samusaran** [IvanJosipovic/BlazorApplicationInsights#243] [Q&A]
Hi, it's not clear to me how I should track dependencies like SQL Queries, to achieve the same behaviour of ASP.NET Core.

**Comment by IvanJosipovic** [comment]
Have you tried installing the ASP.Net Core App Insights library and seeing what it picks up?

https://learn.microsoft.com/en-us/azure/azure-monitor/app/asp-net-core
